modules/RTC/TraceablePeerConnection.js

```diff
@@ -2571,6 +2573,26 @@ TraceablePeerConnection.prototype.setSenderVideoConstraints = function(frameHeig
     if (!localVideoTrack || localVideoTrack.isMuted()) {
         return Promise.resolve();
     }
+
+    const nextPromise = this._lastVideoSenderUpdatePromise
+        .finally(() => this._updateVideoSenderParameters(frameHeight, localVideoTrack));
+
+    this._lastVideoSenderUpdatePromise = nextPromise;
+
+    return nextPromise;
+};
+
+/**
+ * Updating video sender parameters as wrapped-up promise to have ability
```

Comment:
I would prefer to move this text as a comment up to line 2577 because it is very specific to Chrome's implementation of get/setParameters. `_updateVideoSenderParameters` configures the video stream and the JSDoc for this method should reflect that.